tdigest/merging_digest.go

```diff
@@ -335,6 +337,16 @@ func (td *MergingDigest) Max() float64 {
 func (td *MergingDigest) Count() float64 {
 	return td.mainWeight + td.tempWeight
 }
+func (td *MergingDigest) ReciprocalSum() float64 {
+	return td.reciprocalSum
+}
+func (td *MergingDigest) Sum() float64 {
+	var s float64
+	for _, cent := range td.Centroids() {
```

Comment:
oh, you want to iterate over td.mainCentroids and td.tempCentroids here. the function is only included for debugging/analysis purposes